Test or TestAccResourceMongoDBAtlasDatabaseUser_withAWSIAMType with

pitakill opened the pull request:
## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Add test case to cover import for AWS db user

Link to any related issue(s):

https://jira.mongodb.org/browse/INTMDB-156

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the Terraform contribution guidelines
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

pitakill committed:
Test or TestAccResourceMongoDBAtlasDatabaseUser_withAWSIAMType with
import

1. Why is this change necessary?
Test for a import is a nice to have

2. How does it address the issue?
Test the function with the import for a matching scenario

3. What side effects does this change have?
None known

leofigy commented on mongodbatlas/resource_mongodbatlas_database_user_test.go:
```diff
@@ -148,6 +148,43 @@ func TestAccResourceMongoDBAtlasDatabaseUser_withAWSIAMType(t *testing.T) {
+	var (
+		dbUser       matlas.DatabaseUser
+		resourceName = "mongodbatlas_database_user.test"
+		username     = "arn:aws:iam::358363220050:user/mongodb-aws-iam-auth-test-user"
```
this user is in our account right?

leofigy replied:
Hi @pitakill , we could just add an env variable i.e  os.Getenv("TEST_DB_USER_IAM_ARN"), and if not set just set arn:aws:iam::358363220050:user/mongodb-aws-iam-auth-test-user, in case somebody does a clean-up in the innovation account we just update the env variable.

pitakill replied:
Got it! :-)

pitakill committed:
username set in env var

1. Why is this change necessary?
Modular testing

2. How does it address the issue?
Make the username variable in that particular test dependentant on a env
var

3. What side effects does this change have?
None

leofigy reviewed (approved):
LGTM

themantissa reviewed (approved):
LGTM, thank you!
